static/css/around-map.css

```diff
@@ -335,6 +335,15 @@
   color: #297ddb;
 }
 
+.user-avatar {
```

Comment:
Why `.user-avatar` and not `.avatar`?
Do we have non user avatars?

Reply:
I would made a difference because `.avartar` is already use for the avatar in the header.

Reply:
I suppose they share a lot in common then?
For the specificities can't we have `header .avatar` and `.profile .avatar` or something?

Reply:
I already `header .avatar` so no conflict possible

Reply:
Ok, I see what you mean. It's basically the icon of the user on the map. Much different then.